Makefile

```diff
@@ -31,9 +32,10 @@ config:
 	gh workflow run \
 	  -R cdcgov/cfa-config-generator run-workload.yaml  \
 	  -f disease=all \
-	  -f state=all
+	  -f state=all \
+	  -f job_id=$(JOB)
 
-run-batch:
+run-batch: config
```

Comment:
So this calls `config` immediately before running `run-batch`? That's so nice!

Reply:
Yep! Making use of `make` dependencies: https://bytes.usc.edu/cs104/wiki/makefile